### PR TITLE
docs(deploy): add some note for databend-hdfs deploy

### PR DESCRIPTION
**Pull request by PsiACE**

### CODE REVIEW

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 1]
```diff
@@ -541,6 +570,17 @@ d. Open a terminal window and navigate to the folder `/usr/local/databend/bin`.
...
 e. Run the following command to start the Query node:
 
+:::note
+If you need to use HDFS as the storage backend for Stage or Databend itself, please set the following environment variables correctly to ensure proper invocation:
+
+```bash
+export JAVA_HOME=/path/to/java
+export LD_LIBRARY_PATH=${JAVA_HOME}/lib/server:${LD_LIBRARY_PATH}
+export HADOOP_HOME=/path/to/hadoop
+export CLASSPATH=/all/hadoop/jar/files
```
Comment (PsiACE) [inline docs/doc/10-deploy/02-deploying-databend.md:580]: for example:

```
export LD_LIBRARY_PATH={$JAVA_HOME}/lib/server/
export JAVA_HOME=/usr/lib/jvm/java-21-jdk
export HADOOP_HOME={$HOME}/hadoop-3.3.6
export CLASSPATH=$(find $HADOOP_HOME -iname "*.jar" | xargs echo | tr ' ' ':')
```

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 2]
```diff
@@ -163,6 +163,16 @@ f. In the MinIO Console, create a bucket named `databend`.
 
 </TabItem>
 
+<TabItem value="HDFS" label="HDFS">
+
+Before deploying Databend, make sure you have successfully set up your Hadoop environment, and the following tasks have been completed:
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:168]:
```suggestion
Before deploying Databend, make sure you have successfully set up your Hadoop environment, and completed the following tasks:
```

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 3]
```diff
@@ -163,6 +163,16 @@ f. In the MinIO Console, create a bucket named `databend`.
...
+
+- Your system already has an Java SDK installed with JVM support.
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:170]:
```suggestion
- Your system already has a Java SDK installed with JVM support.
```

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 4]
```diff
@@ -163,6 +163,16 @@ f. In the MinIO Console, create a bucket named `databend`.
...
+- Get the name node URL for connecting to HDFS.
+- You have already downloaded the Hadoop release in your system and can access the JAR packages within it.
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:172]:
```suggestion
- You have already downloaded the Hadoop release to your system, and you can access the JAR packages in the  release.
```

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 5]
```diff
@@ -184,6 +194,14 @@ a. Create a folder named `databend` in the directory `/usr/local`.
 
 b. Download and extract the latest Databend release for your platform from [GitHub Release](https://github.com/datafuselabs/databend/releases):
 
+:::note
+If you need to use HDFS as the storage backend for Stage or Databend itself, please choose a release with "hdfs" in its name, for example:
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:198]:
```suggestion
To use HDFS as the storage backend, download a release with a file name formatted as `databend-hdfs-${version}-${target-platform}.tar.gz`.
```

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 6]
```diff
@@ -184,6 +194,14 @@ a. Create a folder named `databend` in the directory `/usr/local`.
...
+
+```bash
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:200]: remove this code snippet

File: docs/doc/10-deploy/02-deploying-databend.md [hunk 7]
```diff
@@ -541,6 +570,17 @@ d. Open a terminal window and navigate to the folder `/usr/local/databend/bin`.
 
 e. Run the following command to start the Query node:
 
+:::note
+If you need to use HDFS as the storage backend for Stage or Databend itself, please set the following environment variables correctly to ensure proper invocation:
```
Comment (soyeric128) [inline docs/doc/10-deploy/02-deploying-databend.md:574]:
```suggestion
When using HDFS as the storage backend, ensure to set the following environment variables:
```

File: docs/doc/10-deploy/02-deploying-databend.md (hunk 1 above)
Comment (soyeric128) [reply docs/doc/10-deploy/02-deploying-databend.md:580]: It would be better if adding this example below the environment variables list.

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**